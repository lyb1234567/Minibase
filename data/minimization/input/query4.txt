Q(x) :- R(x,y), R(x,'b'), R('a','b'), R(u,'c'), R(u,v), S('a','c','d')