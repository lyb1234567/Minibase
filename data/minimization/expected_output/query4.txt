Q(x) :- R(x, 'b'), R('a', 'b'), R(u, 'c'), S('a', 'c', 'd')